databuilder/databuilder/task/search/search_metadata_to_elasticsearch_task.py

```diff
@@ -153,6 +152,8 @@ def run(self) -> None:
             # create index
             LOGGER.info(f"Creating ES index {self.elasticsearch_new_index}")
             index = Index(name=self.elasticsearch_new_index, using=self.elasticsearch_client)
+            doc_mapping = self.document_mapping._doc_type.mapping.meta(name='_meta', params={'version': 2})
```

Comment:
I believe the supported way is:
https://elasticsearch-dsl.readthedocs.io/en/latest/persistence.html#class-meta-options

see also https://github.com/elastic/elasticsearch-dsl-py/blob/master/elasticsearch_dsl/document.py#L95